src/Application/UI/Multiplier.php

```diff
@@ -19,6 +19,9 @@ class Multiplier extends PresenterComponent
 	private $factory;
 
 
+	/**
```

Comment:
When it is not here, PhpStorm shows "Expected Nette\ComponentModel\IContainer got \Closure" warning. And it marks code as wrong, which is very unpleasant, since properly working code should not be marked as bad one.